client/views/room/Room/RoomNotFound/RoomNotFound.tsx

```diff
@@ -0,0 +1,48 @@
+import { States, StatesIcon, StatesTitle, StatesSubtitle, Box } from '@rocket.chat/fuselage';
+import { Meteor } from 'meteor/meteor';
+import React, { ReactElement } from 'react';
+
+import { IRoom } from '../../../../../definition/IRoom';
+import BurgerMenu from '../../../../components/BurgerMenu';
+import TemplateHeader from '../../../../components/Header';
+import { useLayout } from '../../../../contexts/LayoutContext';
+import { useSession } from '../../../../contexts/SessionContext';
+import { useTranslation } from '../../../../contexts/TranslationContext';
+import { RoomTemplate } from '../../components/RoomTemplate/RoomTemplate';
+
+type RoomNotFoundSession = {
+	type: IRoom['t'];
+	name: IRoom['name'];
+	error: Meteor.Error;
+};
+
+const RoomNotFound = (): ReactElement => {
+	const { isMobile } = useLayout();
+	const t = useTranslation();
+	const { name, type } = useSession('roomNotFound') as RoomNotFoundSession;
```

Comment:
As `error` is ignored at `RoomNotFound`, I wonder if we can pass `name` and `type` as props, using the template's data context. @ggazzo thoughts?

Reply:
As we changed the approach here with the design team, we won't use the room data anymore